frontend/test/metabase/scenarios/question/filter.cy.spec.js

```diff
@@ -17,70 +17,203 @@ describe("scenarios > question > filter", () => {
+            name: "12985D",
+          }).then(({ body: { id: DASHBOARD_ID } }) => {
+            cy.log("**-- 4. Add 2 filters to the dashboard --**");
+
+            cy.request("PUT", `/api/dashboard/${DASHBOARD_ID}`, {
+              parameters: [
+                {
+                  name: "Date Filter",
+                  slug: "date_filter",
+                  id: "78d4ba0b",
+                  type: "date/all-options",
+                },
+                {
+                  name: "Category",
+                  slug: "category",
+                  id: "20976cce",
+                  type: "category",
+                },
+              ],
+            });
 
-    // Add Q2 to a dashboard
-    cy.findByText("Yes please!").click();
-    cy.findByText("Orders in a dashboard").click();
+            cy.log("**-- 5. Add nested card to the dashboard --**");
 
-    // Add two dashboard filters
-    cy.get(".Icon-filter").click();
-    cy.findByText("Time").click();
-    cy.findByText("All Options").click();
-    cy.findAllByText("Select…")
-      .last()
-      .click();
-    cy.findByText("Created At").click();
+            cy.request("POST", `/api/dashboard/${DASHBOARD_ID}/cards`, {
+              cardId: Q2_ID,
+            }).then(({ body: { id: DASH_CARD_ID } }) => {
+              cy.log(
+                "**-- 6. Connect dashboard filters to the nested card --**",
+              );
 
-    cy.get(".Icon-filter").click();
-    cy.findByText("Other Categories").click();
-    cy.findAllByText("Select…")
-      .last()
-      .click();
-    popover().within(() => {
-      cy.findByText("Category").click();
+              cy.request("PUT", `/api/dashboard/${DASHBOARD_ID}/cards`, {
+                cards: [
+                  {
+                    id: DASH_CARD_ID,
+                    card_id: Q2_ID,
+                    row: 0,
+                    col: 0,
+                    sizeX: 10,
+                    sizeY: 8,
+                    series: [],
+                    visualization_settings: {},
+                    // Connect both filters and to the card
+                    parameter_mappings: [
+                      {
+                        parameter_id: "78d4ba0b",
+                        card_id: Q2_ID,
+                        target: [
+                          "dimension",
+                          ["field-id", PRODUCTS.CREATED_AT],
+                        ],
+                      },
+                      {
+                        parameter_id: "20976cce",
+                        card_id: Q2_ID,
+                        target: ["dimension", ["field-id", PRODUCTS.CATEGORY]],
+                      },
+                    ],
+                  },
+                ],
+              });
+            });
+            cy.visit(`/dashboard/${DASHBOARD_ID}`);
+          });
+        });
+      });
+
+      cy.get("fieldset")
+        .last()
+        .within(() => {
+          cy.findByText("Category").click();
+        });
```

Comment:
Replace with `cy.findByPlaceholderText("Category").click();` ?

Reply:
Yeah, I thought so as well. For whatever reason, that doesn't work in this scenario. This "Category" is within `<span>` instead of it being a placeholder in an input field.

Why? I have no idea.

Reply:
Because there's a date filter too